shared-files/pmd-ruleset.xml

```diff
@@ -31,4 +32,15 @@
     <rule ref="category/java/multithreading.xml">
         <exclude name="UseConcurrentHashMap"/>
     </rule>
+    <rule ref="category/java/codestyle.xml/MethodNamingConventions">
+        <properties>
+            <property name="junit5TestPattern" value="[a-z]([a-zA-Z0-9]*[a-zA-Z]_?)*[a-zA-Z0-9]*" />
```

Comment:
I think we should take about the naming convention for test methods again.
I encourage the use of nested for each method.
This way we can also alter the method naming, because we no longer need the method name of the method to be tested to be included.

Reply:
need to add the rule to the refarch-tools